transport/src/main/java/io/netty5/channel/ChannelOutboundInvoker.java

```diff
@@ -88,6 +88,25 @@ public interface ChannelOutboundInvoker {
      */
     Future<Void> close();
 
+    /**
+     * Request shutdown one direction of the {@link Channel} and notify the {@link Future} once the operation completes,
+     * either because the operation was successful or because of an error.
+     * <p>
+     * After it the direction was shutdown it either will not produce any inbound data anymore or it will not be
+     * possible to write data anymore (depending on the {@link ChannelShutdownDirection} that was given).
```

Comment:
```suggestion
     * When completed, the channel will either not produce any inbound data anymore, or it will not be
     * possible to write data anymore, depending on the given {@link ChannelShutdownDirection}.
```